src/wavm/io.cpp

```diff
@@ -104,7 +104,7 @@ WAVM_DEFINE_INTRINSIC_FUNCTION(wasi,
                                I32 fdFlags,
                                I32 resFdPtr)
 {
-
+    PROF_START(PathOpen)
```

Comment:
Just a reminder if we really want to merge in these profiling macros. Happy with both.